src/content/configuration/resolve.md

```diff
@@ -112,6 +113,8 @@ The following table explains other cases:
 
 `/abc/node_modules` may resolve in `/node_modules` too.
 
+W> `resolve.alias` will takes precedence over other module resolutions.
```

Comment:
It can either be `X will take precedence over Y` or `X takes precedence over Y` but not both (will, takes).
```suggestion
W> `resolve.alias` takes precedence over other module resolutions.
```